### PR TITLE
Third round of cleanups

### PR DESCRIPTION
**Pull request by Myoldmopar**
Miscellaneous things I found while getting more and more familiar with the codebase.  I don't anticipate doing any more of these, the code looks great.  Next step is carving out the pieces we need when we convert to C++.

### CODE REVIEW

File: glhe/profiles/constant_flow.py [hunk 1]
```diff
@@ -17,4 +17,4 @@ def simulate_time_step(self, inputs: SimulationResponse) -> SimulationResponse:
         return SimulationResponse(inputs.time, inputs.time_step, self.flow_rate, inputs.temperature)
 
     def report_outputs(self) -> dict:
-        return {'{:s}:{:s}:{:s}'.format(self.Type, self.name, ReportTypes.FlowRate): float(self.flow_rate)}
+        return {f"{self.Type}:{self.name}:{ReportTypes.FlowRate}": float(self.flow_rate)}
```
Comment (Myoldmopar) [inline glhe/profiles/constant_flow.py:20]: f

File: glhe/profiles/external_base.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ def read_csv_and_compute_delta_t(file_path: Path, col_num: int) -> [list, list]:
         last_timestamp = None
 
         # Open CSV file and read data
-        with open(file_path, 'r') as csvfile:
+        with file_path.open('r') as csvfile:
```
Comment (Myoldmopar) [inline glhe/profiles/external_base.py:24]: Made it a little more pathlibable.

File: glhe/profiles/external_flow.py [hunk 3]
```diff
@@ -11,12 +13,9 @@ class ExternalFlow(ExternalBase, SimulationEntryPoint):
...
     def __init__(self, inputs: dict, ip: InputProcessor):
 
-        if 'column' in inputs:
-            col_num = inputs['column']
-        else:
-            col_num = 1
-
-        ExternalBase.__init__(self, inputs['path'], col_num=col_num)
+        col_num = inputs.get('column', 1)
+        p = Path(inputs['path']).resolve()
```
Comment (Myoldmopar) [inline glhe/profiles/external_flow.py:17]: More pathlib support.

File: studies/aggregation/scripts/Pareto_Investigations.ipynb
Comment (Myoldmopar) [inline studies/aggregation/scripts/Pareto_Investigations.ipynb:1]: Are these output files?  I'm not sure how they got modified.  I didn't open any of them.

**Review by Myoldmopar — COMMENTED** [review]
Minor changes only, ready for more intense work in here soon.

File: studies/aggregation/scripts/Pareto_Investigations.ipynb
Comment (mitchute) [reply studies/aggregation/scripts/Pareto_Investigations.ipynb:1]: None of these notebooks should be output files. Feel free to revert the diffs.

**Comment by Myoldmopar** [comment]
OK, that's better.  I'm still not sure what modified those ipynb files.  Maybe I used an IDE refactor function and it thought it needed to update those in there.  :shrug: 

Either way, this is down to just some Python cleanup changes and should be able to drop right in.  I'm done with cleanup here, ready to start cutting it down to the minimal version for translation.

**Comment by Myoldmopar** [comment]
I don't see a way to approve my own changes, so I'll just leave that to you @mitchute.  Review/comment/approve/merge/delete whatever you'd like.  I'll move onto the other project...

**Review by mitchute — APPROVED** [review]

**Pull request merged**